src/synthetix/StakingRewards.sol

```diff
@@ -135,6 +137,11 @@ contract StakingRewards is IStakingRewards, Pausable, ReentrancyGuard {
 
     /* ========== RESTRICTED FUNCTIONS ========== */
 
+    /// @dev Before calling this function, the caller should send at least `reward` tokens to the contract.
```

Comment:
Neat - I wouldn't put the @dev natspec tag, as natspec is not used in other places in this contract.

Reply:
Updated.